src/visualizers/SeqColor.ts

```diff
@@ -0,0 +1,266 @@
+    private boxIsShow = false
+    private primeNum: number[] = []
+    private countPrime = 0
+    private firstDraw = true
+
+    initialize(sketch: p5, seq: SequenceInterface) {
+        super.initialize(sketch, seq)
+        this.sketch = sketch
+        this.seq = seq
+        this.currentNum = seq.first
+        this.X = 0
+        this.Y = 0
+        this.ready = true
+    }
+
+    setup() {
+        this.sketch.background('black')
+        this.sketch.colorMode(this.sketch.HSB, 360, 100, 100)
+        this.sketch.frameRate(30)
+
+        // Set position of the circle
+        this.X = this.canvasSizeX / this.n + 30
+        this.Y = this.canvasSizeX / this.n + 50
+
+        // Obtain all prime numbers from the sequence
+        for (let i = this.seq.first; i < this.n; i++) {
+            const checkCurrentPrime = Number(this.seq.getElement(i))
+            if (
+                this.isPrime(checkCurrentPrime)
+                && !this.primeNum.includes(checkCurrentPrime)
+            ) {
+                this.primeNum.push(checkCurrentPrime)
+                this.countPrime += 1
+            }
+        }
+    }
+
+    draw() {
+        if (this.firstDraw == true && this.currentNum < this.n) {
+            const currentElement = Number(
+                this.seq.getElement(this.currentNum++)
+            )
+            this.drawCircle(currentElement)
+            this.showCircleLabel(currentElement)
+
+            // Check if drawing finished
+            if (this.currentNum >= this.n) {
+                this.firstDraw = false
+            }
+            this.sketch.noStroke()
+        } else {
+            // Monitor keyboard events after finishing drawing
+            // Bug: multiple clicks detected when only one click happened
+            this.keyboardEvents()
+        }
+    }
+
+    keyboardEvents() {
+        // Show description box when the lft arrow key is pressed
+        if (this.sketch.keyIsDown(this.sketch.LEFT_ARROW)) {
+            if (this.boxIsShow == true && this.subG != null) {
+                this.boxIsShow = false
+                this.undrawBox()
+            } else {
+                this.boxIsShow = true
+                this.drawBox()
+            }
+        }
+        // Refresh
+        if (
+            this.sketch.keyIsDown(this.sketch.RIGHT_ARROW)
+            && this.firstDraw == false
+        ) {
+            this.firstDraw = true
+        }
+    }
+
+    drawCircle(numberNow: number) {
+        this.sketch.ellipseMode(this.sketch.RADIUS)
+        let radius = 50
+        let bright = 0
+
+        for (let x = this.n; x >= 2; x--) {
+            // Calculate the difference
+            //  between 1/(number)^n and 1/(number)^(n-1)
+            const diff = Math.abs(
+                Math.log(
+                    1 / Math.pow(Number(numberNow), x - 1)
+                        - Math.log(1 / Math.pow(Number(numberNow), x))
+                )
+            )
+
```

Comment:
What about negative terms?  Right now it draws black.  I suggest you just take the absolute values of terms, so that the sequences will converge.  If you do this, you should also document this behaviour in the documentation above.